### PR TITLE
feat(cni): enable cni and add deploy scripts

### PR DESCRIPTION
**Pull request by wu-cl**

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -63,10 +63,13 @@ jobs:
       - name: build and import image
         run: |
           make images
-          docker save everoute/release | sudo k3s ctr images import -
+          docker tag everoute/release everoute/release:latest
+          docker save everoute/release:latest | sudo k3s ctr images import -
```
Comment (zwtop) [inline .github/workflows/ci.yaml:67]: why retag from everoute/release to everoute/release:latest ？

File: Makefile [hunk 2]
```diff
@@ -18,12 +18,19 @@ docker-generate:
 	$(eval WORKDIR := /go/src/github.com/everoute/everoute)
 	docker run --rm -iu $$(id -u):$$(id -g) -w $(WORKDIR) -v $(CURDIR):$(WORKDIR) everoute/generate make generate
 
+images-test:
+	docker build -f build/images/release/Dockerfile -t harbor.smartx.com/everoute/dev .
+	docker push harbor.smartx.com/everoute/dev:latest
```
Comment (zwtop) [inline Makefile:23]: nit:
`image-test` is unnecessary, and maybe it's not a graceful solution to use private registry here?

File: build/images/release/Dockerfile [hunk 3]
```diff
@@ -4,17 +4,34 @@ FROM golang:1.15 as builder
...
+ADD . $GOPATH/src/everoute
+
 # Build
-COPY . $GOPATH/src/everoute
-RUN mkdir -p /opt/everoute/bin && \
-    make bin && \
-    cp $GOPATH/src/everoute/bin/* /opt/everoute/bin/
+#RUN --mount=type=cache,target=/root/.cache/go-build go build -o bin/everoute-controller cmd/everoute-controller/main.go
+#RUN --mount=type=cache,target=/root/.cache/go-build go build -o bin/everoute-agent cmd/everoute-agent/*.go
+#RUN --mount=type=cache,target=/root/.cache/go-build go build -o bin/everoute-cni cmd/everoute-cni/*.go
```
Comment (zwtop) [inline build/images/release/Dockerfile:22]: nit:
 remove deprecated codes

File: .github/workflows/ci.yaml [hunk 4]
```diff
@@ -63,10 +63,13 @@ jobs:
...
 
       - name: make kubernetes deploy test
-        run: make deploy-test
+        run: |
+          sed -i 's/# imagePullPolicy: IfNotPresent/imagePullPolicy: IfNotPresent/g' deploy/everoute.yaml
```
Comment (zwtop) [inline .github/workflows/ci.yaml:71]: why remove `imagePullPolicy: IfNotPresent` in deploy/everoute.yaml and use sed here?

File: .github/workflows/ci.yaml (hunk 1 above)
Comment (wu-cl) [reply .github/workflows/ci.yaml:67]: image in controller.yaml & agent.yaml were set to everoute/release:latest. Only latest tag will re-pull during each installation.

File: build/images/release/Dockerfile (hunk 3 above)
Comment (wu-cl) [reply build/images/release/Dockerfile:22]: It's ok.

File: Makefile (hunk 2 above)
Comment (wu-cl) [reply Makefile:23]: These test related op will be removed.

File: .github/workflows/ci.yaml (hunk 4 above)
Comment (wu-cl) [reply .github/workflows/ci.yaml:71]: ditto. for test convenience, It will be removed.

**Review by KeyongSun — APPROVED** [review]

File: pkg/agent/proxy/route.go [hunk 5]
```diff
@@ -176,6 +176,13 @@ func UpdateIptables(nodeList corev1.NodeList, thisNode corev1.Node) {
 		return
 	}
 
+	// set FROWARD in filter to accept
```
[the remark this answers is not in the document]
Comment (wu-cl) [reply pkg/agent/proxy/route.go]: done

**Pull request merged**